saleor/graphql/translations/mutations.py

```diff
@@ -24,6 +26,11 @@ def check_permissions(cls, context):
 
     @classmethod
     def perform_mutation(cls, _root, info, **data):
+        if "id" in data and not data["id"]:
```

Comment:
`if not data.get("id"):`?

Reply:
Comparison: https://gist.github.com/NyanKiyoshi/b221de2be1b0cf2ea89d3de5bf044809

```

In [10]:

data1 = {"id": "abcde", "data": []}
data2 = {"data": []}

In [11]:

%timeit "id" in data1 and not data1["id"]

62.5 ns ± 0.57 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [12]:

%timeit not data1.get("id")

57.6 ns ± 0.674 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [13]:

%timeit "id" in data2 and not data2["id"]

29.3 ns ± 0.286 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [14]:

%timeit not data2.get("id")

54.8 ns ± 0.795 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

```